tox.ini

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
```

Comment:
I normally let Travis do the job. And in case I detect problems related to python version, i'll just create two virtualenvs. It's rare that I continuesly want to run all tests on both python versions because of the time it takes